.circleci/config.yml

```diff
@@ -1,34 +1,20 @@
-# Python CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-python/ for more details
-#
 version: 2
 jobs:
   build:
     docker:
-      # specify the version you desire here
-      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
       - image: python:3.4-jessie
-
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/postgres:9.4
-
     working_directory: ~/robot
 
     steps:
       - checkout
 
-      # Download and cache dependencies
       - restore_cache:
           keys:
           - v1-dependencies-{{ checksum "requirements.txt" }}
           - v1-dependencies-{{ checksum "robotd/requirements.txt" }}
```

Comment:
I'm not sure this line can ever do anything, though even if it does I don't think we want that to happen. Remove it?

Reply:
Hmm, yea, agreed. Not sure what the use of that line really is